### DISCUSSION TITLE
Using MosaicJSON stored locally

### DISCUSSION THREAD

**Discussion by giswqs** [developmentseed/titiler#590] [Q&A] [answered]
Is it possible to use a MosaicJSON stored locally? Imagine that a user creates the MosiacJSON file locally and want to visualize the mosaic using TiTiler without having to upload the MosiacJSON file to the web. 

```python
r = httpx.get(
    f"{titiler_endpoint}/mosaicjson/tilejson.json",
    params={
        # "url": "https://gist.githubusercontent.com/vincentsarago/c6ace3ccd29a82a4a5531693bbcd61fc/raw/e0d0174a64a9acd2fb820f2c65b1830aab80f52b/NOAA_Nashville_Tornado.json" 
        "url": "NOAA_Nashville_Tornado.json"
    }
).json()
print(r)
```

> {'detail': "[Errno 2] No such file or directory: 'NOAA_Nashville_Tornado.json'"}

**Comment by vincentsarago** [comment, marked as answer]
sadly it's not possible. We also don't offer hosting of mosaicjson because we didn't want to care about authentification (to make sure someone is not filling up our S3 bucket with some files)
